adam-core/src/test/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDSuite.scala

```diff
@@ -106,6 +106,16 @@ class AlignmentRecordRDDSuite extends ADAMFunSuite {
     assert(coverage.rdd.filter(r => r.start == 30).first.count == pointCoverage)
   }
 
+  sparkTest("test filterByOverlappingRegions") {
+    val inputPath = resourcePath("artificial.sam")
```

Comment:
You'll want to change `resourcePath("artificial.sam")` to `testFile("artificial.sam")` and then this should compile.